### PR TITLE
[blend2d] Update to beta_2021-03-17

### PR DESCRIPTION
**Pull request by ZeeWanderer**
Updated blend2d to beta_2021-03-17. Updated local asmjit to 2021-03-19.
versioning scheme changed from `version-string` to `version-date`
Fixed facepalm `BLEND2D_BUILD_NO_JIT` usage in portfile. Probably forgot to rename after blend2d changed variable naming to `BLEND2D_NO_JIT`.

### CODE REVIEW

**Review by JonLiu1993 — APPROVED** [review]

File: ports/blend2d/portfile.cmake [hunk 1]
```diff
@@ -21,8 +21,8 @@ if(NOT BLEND2D_BUILD_NO_JIT)
   vcpkg_from_github(
     OUT_SOURCE_PATH ASMJIT_SOURCE_PATH
```
Comment (ras0219-msft) [inline ports/blend2d/portfile.cmake:22]: Noting here that asmjit is available as a vcpkg port: https://github.com/microsoft/vcpkg/blob/master/ports/asmjit/CONTROL

By policy we would _really_ like this to stop being embedded here, however that won't block this update PR.

File: ports/blend2d/vcpkg.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "blend2d",
-  "version-string": "beta_2021-01-02",
+  "version-string": "beta_2021-03-17",
```
Comment (ras0219-msft) [inline ports/blend2d/vcpkg.json:3]:
```suggestion
  "version-date": "2021-03-17",
```
It looks like this is a rolling release, so we should use a date scheme

File: versions/b-/blend2d.json [hunk 3]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "a73748cae8a849c48f8a749e2ff9b3c85a572d43",
+      "version-string": "beta_2021-03-17",
```
Comment (ras0219-msft) [inline versions/b-/blend2d.json:5]:
```suggestion
      "version-date": "2021-03-17",
```

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
LGTM after the version scheme change, thanks for the PR!

File: ports/blend2d/portfile.cmake (hunk 1 above)
Comment (ZeeWanderer) [reply ports/blend2d/portfile.cmake:22]: blend2d depends on asmjit CMakeLists directly. I could try to patch it but that would complicate this port by a lot.
https://github.com/blend2d/blend2d/blob/master/CMakeLists.txt#L373

**Comment by ras0219-msft** [comment]
Thanks!

**Pull request merged**